spec/models/ability_spec.rb

```diff
@@ -485,13 +477,6 @@
       end
     end
 
-    context 'can view, create and change any external_users details' do
```

Comment:
ditto here on having a  negative test.

could use a [shared example](https://relishapp.com/rspec/rspec-core/v/3-8/docs/example-groups/shared-examples) with parameters to have positive and negative versions, but not a blocker

```
it_behaves_like 'a provider manager', false
it_behaves_like 'a provider manager', true
```